src/components/LibrariesListContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import LibrariesList from "./LibrariesList";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibrariesData, LibraryData } from "../interfaces";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+
+export interface LibrariesListContainerContext {
+  store: Store<State>;
```

Comment:
@aslagle There was a store issue where a component could not readily connect to the store. I had this issue before and I told Adriana to create this top level component to use the store and pass it down so that the child component can connect to the store. I think we can eventually do without this component since the store in this app only has one value. We'll revisit this issue later (not sure if it actually is an issue though).

Reply:
A lot of online tutorials suggest using this pattern (for whatever that's worth...).

Reply:
Someone should review the latest redux best practices. When we started the recommendation was to connect top-level components to the store and pass everything down to other components in props, but I think at some point they changed to recommend that you connect individual components that need something from the store. That was a while ago too so I don't know what's suggested now.

Reply:
I can pick it up because I don't have too much to work on now.